client/src/questions-n-answers/qNa.jsx

```diff
@@ -117,7 +119,7 @@ class QuestionsNAnswers extends React.Component {
             } else {
               currentClass = this.state.questionHide
             }
-            return <QuestionsAndAnswers key={index} currentI={index} showButton={this.showButton} lastI={this.state.lastIndex} answerScroll={this.state.answerScroll} questionCount={this.state.questionClickCount} answerCount={this.state.answerClickCount} classname={currentClass} data={question}/>
+            return <QuestionsContainer key={index} currentI={index} showButton={this.showButton} lastI={this.state.lastIndex} answerScroll={this.state.answerScroll} questionCount={this.state.questionClickCount} answerCount={this.state.answerClickCount} classname={currentClass} data={question}/>
```

Comment:
This could be styled to be a little more readable if you had properties vertically instead of horizontally listed. Bit of a preference thing, but I'd maybe lean towards having something like:
<QuestionsContainer 
  key={index}
  currentI={index}
  showButton={this.showButton}
  etc={etc}
/>